lib/round.rb

```diff
@@ -0,0 +1,47 @@
+class Round
+    attr_reader :deck
+    attr_reader :turns
+
+    def initialize(deck)
+        @deck = deck
+        @turns = []
+        @number_correct = 0
+    end
+
+
+    def current_card 
+        @deck.cards[@turns.count]
+    end
+
+    def take_turn(guess)
+        new_turn = Turn.new(guess, current_card)
+        @turns << new_turn
+        if new_turn.correct?
+            @number_correct += 1
+        end
+        new_turn
+    end
+    
+    def number_correct
+        @number_correct
+    end
+    
+    def number_correct_by_category(category)
+        @turns.count do |turn|
+            turn.card.answer == turn.guess && turn.card.category == category
+        end
+    end
+
+
+    def percent_correct
+        @number_correct.to_i / @turns.count.to_i * 100.0
+    end
+
+    def percent_correct_by_category(category)
+        total_turns_by_category = @turns.count do |turn|
+            turn.card.category == category 
+        end
+        100*number_correct_by_category(category).to_i / total_turns_by_category.to_i
+    end
```

Comment:
If we were to assume that this method gets called at the end of the game, when all cards in the deck have been guessed on, you could use the `cards_in_category` method in your deck class to get the number of cards in that category within the deck. I do like that the way you have your method written does make this calculation dynamic for however many cards have been guessed upon at any given moment. So, truly no preference on my end, just wanted to point it out in case you didn't think of that ;)